Cartfile.resolved

```diff
@@ -1,4 +1,4 @@
-binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" "3.4.2"
+binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" "3.5.0"
```

Comment:
Can you also update https://github.com/mapbox/MapboxNavigation.swift/blob/master/Cartfile#L1